main.go

```diff
@@ -2,15 +2,22 @@ package main
 
 import (
 	"encoding/json"
-	"fmt"
+	"github.com/eldeal/skills/config"
+	"github.com/gorilla/mux"
 	"io/ioutil"
-	"log"
 	"net/http"
+	"os"
 
-	"github.com/gorilla/mux"
+	"github.com/ONSdigital/log.go/log"
 )
 
 func main() {
+	// Get Config
```

Comment:
when using the `log.go` library, it's _essentially_ a requirement to add a line like this:
```
  log.Namespace = "books"
```

cf https://github.com/ONSdigital/dp-dataset-api/blob/develop/main.go#L27